javascripts/buzz.js

```diff
@@ -224,6 +224,10 @@ app.buzz = {
 
           app.buzz.infiniteScrollCalled = true;
           var from = $('.buzz-item').length + 1;
+        
+          if($('.buzz-item').length == 4){
```

Comment:
I think you have just moved the gap out to the next request. I suspect you are now getting items:

    0-3
    4-11
    13-20

Where has item 12 gone?